src/modules/csm/csm.py

```diff
@@ -119,16 +119,20 @@ def build_report(self, blockstamp: ReferenceBlockStamp) -> tuple:
                 shares[no_id] += amount
 
         tree = self.make_tree(shares)
-        cid: CID | None = None
+        tree_cid: CID | None = None
```

Comment:
useless line

Reply:
You can't use `tree_cid if tree else prev_cid` because `tree_cid` should be declared anyway.